bindings/python/test/environment/ephemeris/test_celestial_analytical.py

```
@@ -0,0 +1,38 @@
+    ]
+)
+def celestial_analytical(request) -> CelestialAnalytical:
+    return CelestialAnalytical(request.param)
+
+
+class TestCelestialAnalytical:
+    def test_constructor_success(self, celestial_analytical: CelestialAnalytical):
+        assert celestial_analytical is not None
+
+    def test_constructor_unsupported_type_failure(self):
+        with pytest.raises(Exception):
```

Ruff (0.15.21)

[warning] Do not assert blind exception: `Exception`

(B017)